sql/core/src/main/scala/org/apache/spark/sql/sources/commands.scala

```diff
@@ -127,8 +127,9 @@ private[sql] case class InsertIntoHadoopFsRelation(
     val needsConversion = relation.needConversion
     val dataSchema = relation.dataSchema
 
+    writerContainer.driverSideSetup()
```

Comment:
This call only initializes and prepares the job, any exception thrown from here shouldn't cause `abortJob()` to be called.